treeherder/webapp/api/performance_serializers.py

```diff
@@ -178,9 +179,18 @@ class Meta:
         ]
 
 
+class PerformanceTagSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(read_only=True)
+
+    class Meta:
+        model = PerformanceTag
+        fields = ['id', 'name']
+
+
 class PerformanceAlertSummarySerializer(serializers.ModelSerializer):
     alerts = PerformanceAlertSerializer(many=True, read_only=True)
     related_alerts = PerformanceAlertSerializer(many=True, read_only=True)
+    performance_tags = PerformanceTagSerializer(many=True, required=False)
```

Comment:
Shouldn't we be using a `SlugRelatedField` here? Or have I missed something?

Reply:
Yes, we should, the pr is updated now